docs/item-service.md

```diff
@@ -0,0 +1,35 @@
+# Item service
+
+Manages the description and auction status (created, auction, completed, cancelled) of an item.
+
+## Queries handled
+
+* **getItem** - Gets an item by an ID.
+* **getItemsForUser** - Gets a list of items in a provided status that are owned by a given user.
+
+## Events emitted
+
+* **AuctionStarted** (public) - When the auction is started, in response to **startAuction**.
+* **ItemUpdated** (private) - When user editable fields on an item are updated in response to **createItem**.
```

Comment:
Is this page copy paster from the `online-auction-java`? 

Note to self: I'm not sure the scala version currently complies with this docs. We'll have to review later... Since the `Edit Item` feature is being worked on in a separate PR we can review this docs when the feature is complete.